micro-frontends/src/next-ui/utils/FormDisplayControl/FormView.js

```diff
@@ -1,9 +1,10 @@
 import axios from "axios";
-import moment from "moment";
 import { ENCOUNTER_BASE_URL } from "../../constants";
 import { build } from "../FormDisplayControl/BuildFormView";
+import { defaultDateFormat } from "../../constants";
+import { formatDate } from "../../utils/utils";
 
-export const findByEncounterUuid = async (encounterUuid) => {
+var findByEncounterUuid = async (encounterUuid) => {
```

Comment:
@ayush34957 I've noticed that this particular function is being used in different files. This change will cause issues in other parts where findByEncounterUuid is being imported and used. Specifically, this modification breaks the edit functionality, preventing the edit page from rendering properly. Since var does not export the function, it can no longer be imported in other files.

I hope, to ensure the function remains accessible where it is needed, we need to revert this change.

cc: @binduak

Reply:
Done